ovn-org/ovn-kubernetes
# enable interconnect in helm chart

add `-ic` option to `kind-helm.sh` to enable interconnect
    - if npz is not specified, create single-node zone interconnect
    - if npz is greater than 1, create multi-node zone interconnect

#### What this PR does and why is it needed
support interconnect in Helm chart

#### Which issue(s) this PR fixes
#4371

#### Special notes for reviewers
N/A

#### How to verify it
contrib/kind-helm.sh -ic, should create ovn-k8s with single-node zone interconnect
contrib/kind-helm.sh -ic -wk3 -npz 2, should create ovn-k8s with multi-node zone interconnect
`contrib/kind-helm.sh` only, i.e. without `-ic` option, should create ovn-k8s with interconnect disabled.

#### Details to documentation updates
N/A


#### Description for the changelog
N/A

#### Does this PR introduce a user-facing change?
N/A
